sql/core/src/test/scala/org/apache/spark/sql/execution/arrow/ArrowWriterSuite.scala

```diff
@@ -30,7 +30,11 @@ class ArrowWriterSuite extends SparkFunSuite {
 
   test("simple") {
     def check(dt: DataType, data: Seq[Any], timeZoneId: String = null): Unit = {
-      val schema = new StructType().add("value", dt, nullable = true)
+      val avroDatatype = dt match {
+        case _: YearMonthIntervalType => YearMonthIntervalType()
```

Comment:
Sorry, I didn't get this. Do you always test only `YearMonthIntervalType()`?

Reply:
> Sorry, I didn't get this. Do you always test only `YearMonthIntervalType()`?

We can write different field of YearMonthIntervalType tp arrow, but in arrow's type it only support year_to_month
![image](https://user-images.githubusercontent.com/46485123/123025989-09901400-d40e-11eb-8581-5e9dfe4f9a33.png)
And the IntervalUnit only support year_to_month, 
![image](https://user-images.githubusercontent.com/46485123/123026045-20cf0180-d40e-11eb-940c-24ab76aa9a79.png)

So when we read data from Arrow, it only can be default YearMonthIntervalType()

Reply:
ok. I see. Thanks.